### ISSUE TITLE
Is this network working on CUB dataset?

### ISSUE THREAD

**Issue by jeong-tae** [Charleo85/DeepCar#2]
I am trying to reproduce RACNN network performance with pytorch.
But there isn't details about how to train a APN network.

without pretraining, rank loss doesn't decrease.
i am wonder this code work on CUB to decrease the APN net loss.

**Comment by JUSTDODoDo** [comment]
hello,I am very interested in your work on RA-CNN Implemented  by pytorch, but as a newbie, how do I prepare to train the dataset, I have already got the entire dataset of Compcars, you have not explained clearly in Readme. Training, can you tell the details, thank you!
